assets/js/main.js

```diff
@@ -16,71 +17,86 @@ gallery();
 table();
 burger();
 
-window.addEventListener('scroll', function () {
-    'use strict';
-    if (body.classList.contains('home-template') && body.classList.contains('with-full-cover') && !document.querySelector('.cover').classList.contains('half')) {
+window.addEventListener("scroll", function () {
```

Comment:
This whole file was reformatted for only a few lines added. Can you revert the reformatting/beautifying, please? 
I'd like to keep it in sync with the upstream theme as much as possible, thanks.